src/createServer.js

```diff
@@ -1,9 +1,29 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const requestURL = new URL(
+      req.url.replaceAll('//', '/'),
```

Comment:
The use of `replaceAll('//', '/')` may not be necessary or could lead to unexpected results if the URL contains double slashes intentionally. Consider whether this replacement is needed or if it could be handled differently to avoid altering the URL in unintended ways.